targets/science/include/PinNames.h

```diff
@@ -162,30 +162,23 @@ typedef enum {
   LED_G = PC_2,
   LED_B = PC_3,
 
-  BUTTON_1 = PB_0,
-  BUTTON_2 = PB_1,
-
-  /**** Servo Pins *****/
-
-  SRVO_PWM_1 = PB_2,
-  SRVO_PWM_2 = PB_10,
+  BUTTON_1 = PA_2,
 
   /**** Actuator Pins ****/
   MTR_PWM_1 = PA_6,
-  MTR_DIR_1 = PA_5,
+  MTR_DIR_1 = PC_5,
   MTR_PWM_2 = PA_7,
   MTR_DIR_2 = PC_4,
 
   /**** Limit Switch Pins ****/
-  LIM_SW_1 = PB_4,
-  LIM_SW_2 = PB_3,
-  LIM_SW_3 = PD_2,
-  LIM_SW_4 = PC_12,
+  LIM_SW_1 = PC_15,
+  LIM_SW_2 = PC_14,
+  LIM_SW_3 = PC_13,
+  LIM_SW_4 = PB_9,
 
```

Comment:
Just checked the science rev 2 schematic and looks like a lot of pins are missing? Like LIM_SW_5, SPI pins, servo pins... Could be that the ioc file is out of date, may wanna check this with Wolfie

Reply:
@WolfgangWindholtz Can you verify the science CubeMX file?